tap-snapshots/examples-custom-root-component-index.test.js-TAP.test.js

```diff
@@ -8,25 +8,5 @@
 exports[
   `examples/custom-root-component/index.test.js TAP examples/custom-root-component > page 1`
 ] = `
-<!DOCTYPE html><html>
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui" />
-    
-      <title data-ream-head="true"></title>
-      
-      
-      
-      <link rel="preload" href="/_ream/chunk-vendors.js" as="script"><link rel="preload" href="/_ream/client.js" as="script"><link rel="prefetch" href="/_ream/chunk--root.js"><style data-vue-ssr-id="83b416fe:0">
-h1[data-v-b04334aa] {
-  color: blue;
-}
-</style>
-  </head>
-  <body>
-    <div id="_ream" data-server-rendered="true"><div data-v-b04334aa><h1 data-v-b04334aa>custom root component</h1> <h1 data-v-b04334aa>homepage</h1></div></div>
-    <script>window.__REAM__={"initialData":{}}</script><script src="/_ream/chunk-vendors.js" defer></script><script src="/_ream/client.js" defer></script>
-  </body>
-</html>
+<!doctype html><html><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no,minimal-ui"><title data-ream-head="true"></title><link rel="preload" href="/_ream/chunk-vendors.js" as="script"><link rel="preload" href="/_ream/client.js" as="script"><link rel="prefetch" href="/_ream/chunk--root.js"><style data-vue-ssr-id="83b416fe:0">h1[data-v-b04334aa]{color:#00f}</style></head><body><div id="_ream" data-server-rendered="true"><div data-v-b04334aa><h1 data-v-b04334aa>custom root component</h1><h1 data-v-b04334aa>homepage</h1></div></div><script>window.__REAM__={"initialData":{}}</script><script src="/_ream/chunk-vendors.js" defer="defer"></script><script src="/_ream/client.js" defer="defer"></script></body></html>
```

Comment:
Please keep the existing tests unchanged. They are now unreadable without purpose.

Please add a **new** test instead which tests the new behaviour (or better two, one with default options and one with custom minify options).